Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/IntakeEject.java

```diff
@@ -99,55 +103,98 @@ public void runIntake(double speed) {
 	 * Raises the intake
 	 */
 	public void raiseIntake() {
-		DoubleSolenoid.Value leftSet = Robot.getBool("Intake Left Vertical Solenoid Inverted", false)
-				? DoubleSolenoid.Value.kReverse
-				: DoubleSolenoid.Value.kForward;
-		DoubleSolenoid.Value rightSet = Robot.getBool("Intake Right Vertical Solenoid Inverted", false)
-				? DoubleSolenoid.Value.kReverse
-				: DoubleSolenoid.Value.kForward;
-		leftVerticalSolenoid.set(leftSet);
-		rightVerticalSolenoid.set(rightSet);
+		leftVerticalSolenoid.set(solenoidSet(true, true));
+		rightVerticalSolenoid.set(solenoidSet(false, true));
 	}
 
 	/**
 	 * Lowers the intake
 	 */
 	public void lowerIntake() {
-		DoubleSolenoid.Value leftSet = Robot.getBool("Intake Left Vertical Solenoid Inverted", false)
-				? DoubleSolenoid.Value.kForward
-				: DoubleSolenoid.Value.kReverse;
-		DoubleSolenoid.Value rightSet = Robot.getBool("Intake Right Vertical Solenoid Inverted", false)
-				? DoubleSolenoid.Value.kForward
-				: DoubleSolenoid.Value.kReverse;
-		leftVerticalSolenoid.set(leftSet);
-		rightVerticalSolenoid.set(rightSet);
+		leftVerticalSolenoid.set(solenoidSet(true, false));
+		rightVerticalSolenoid.set(solenoidSet(false, false));
+	}
+
```

Comment:
Fyi, none of the intakes under consideration use pistons to raise/lower the intake, so the associated code can be removed.

Reply:
Delete in latest commit.